content/tutorials/r_google_sheets/index.md

```diff
@@ -1,134 +1,28 @@
 ---
-title: "Read data from google sheet"
-description: "Extracting data driectly from google spreadhseets"
-author: "Stijn Van Hoey"
-date: 2017-01-03
+title: "Read data from Google sheets"
+description: "Extracting data directly from Google spreadsheets"
+author: "Damiano Oldoni"
+date: 2020-10-19
 categories: ["r"]
 tags: ["google", "r", "spreadsheet"]
-output: 
-    md_document:
-        preserve_yaml: true
-        variant: markdown_github
 ---
 
-``` r
-library(googlesheets)
-library(dplyr)
-library(ggplot2)
-```
+## `googlesheets4`
 
-Setup
------
+The R package [googlesheets4](https://googlesheets4.tidyverse.org/) provides the functionality to read and write data from a Google sheet.
 
-The R package [googlesheets](https://github.com/jennybc/googlesheets/blob/master/README.md) provides the functionality to retrieve data from a google sheet. Once the registration as a user is done and the permissions are granted, it enables to read and write to google sheets. Initiation of the authentification can be done with the `gs_auth` command.
+This package is very well documented, so we just have to provide you with the right links:
 
-``` r
-gs_auth()
-```
+- [read sheets](https://googlesheets4.tidyverse.org/articles/articles/read-sheets.html)
```

Comment:
I would refer to the page https://googlesheets4.tidyverse.org/articles/googlesheets4.html here instead.  The page you refer to is already more specific if I'm correct:

> Basic Sheet reading is shown in the Get started article. Here we show how to target a specific (work)sheet or cell range, how to deal with column types, and how to get detailed cell data.

Reply:
Merci @Lien! Indeed! I will change it.

Reply:
Done: https://github.com/inbo/tutorials/pull/204/commits/cf59e8d6dccd8a9814b1df59c219163f5137fa80